### ISSUE TITLE
[BUG] Requirement listing extras misses space

### ISSUE THREAD

**Issue by jakirkham** [conda-incubator/grayskull#260]
**Describe the bug**

When having a dependency like [this]( https://github.com/dask-contrib/dask-sql/blob/2194a75b0dfd6e5f981be031c2f662a41297140f/setup.py#L77 )

```
dask[dataframe,distributed]==2021.10.0
```

Grayskull strips the optional requirements (somewhat understandable since we don't have a way to handle these yet), though it misses adding a space giving us this

```
dask==2021.10.0
```

This also creates a linter error ( https://github.com/conda-forge/dask-sql-feedstock/pull/28#issuecomment-965686708 )

**To Reproduce**

Run `grayskull dask-sql`. Currently based off of `dask-sql` version `2021.11.0`.

**Expected behavior**

Space would be added.

**Outputs**

If applicable, add the output to help explain your problem.

**Environment:**

```
→ grayskull --version
0.9.1
```

<details>

```
→ conda list
# packages in environment at /raid/charlesb/miniconda3/envs/conda-forge:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       1_gnu    conda-forge
alabaster                 0.7.12                     py_0    conda-forge
babel                     2.9.1              pyh44b312d_0    conda-forge
beautifulsoup4            4.10.0             pyha770c72_0    conda-forge
blinker                   1.4                        py_1    conda-forge
boolean.py                3.7                        py_0    conda-forge
brotlipy                  0.7.0           py39h3811e60_1003    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
c-ares                    1.18.1               h7f98852_0    conda-forge
ca-certificates           2021.10.8            ha878542_0    conda-forge
certifi                   2021.10.8        py39hf3d152e_1    conda-forge
cffi                      1.15.0           py39h4bc2ebd_0    conda-forge
chardet                   4.0.0            py39hf3d152e_2    conda-forge
charset-normalizer        2.0.0              pyhd8ed1ab_0    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
conda                     4.10.3           py39hf3d152e_3    conda-forge
conda-build               3.21.6           py39hf3d152e_0    conda-forge
conda-forge-pinning       2021.11.10.13.02.31      hd8ed1ab_0    conda-forge
conda-package-handling    1.7.3            py39h3811e60_1    conda-forge
conda-smithy              3.14.3             pyhd8ed1ab_0    conda-forge
cryptography              35.0.0           py39h95dcef6_2    conda-forge
curl                      7.80.0               h2574ce0_0    conda-forge
deprecated                1.2.13             pyh6c4a22f_0    conda-forge
docutils                  0.16             py39hf3d152e_3    conda-forge
expat                     2.4.1                h9c3ff4c_0    conda-forge
filelock                  3.3.2              pyhd8ed1ab_0    conda-forge
gettext                   0.19.8.1          h73d1719_1008    conda-forge
git                       2.33.1          pl5321hc30692c_0    conda-forge
gitdb                     4.0.9              pyhd8ed1ab_0    conda-forge
gitpython                 3.1.24             pyhd8ed1ab_0    conda-forge
glob2                     0.7                        py_0    conda-forge
grayskull                 0.9.1              pyhd8ed1ab_0    conda-forge
icu                       69.1                 h9c3ff4c_0    conda-forge
idna                      3.1                pyhd3deb0d_0    conda-forge
imagesize                 1.3.0              pyhd8ed1ab_0    conda-forge
isodate                   0.6.0                      py_1    conda-forge
jinja2                    3.0.3              pyhd8ed1ab_0    conda-forge
krb5                      1.19.2               hcc1bbae_3    conda-forge
ld_impl_linux-64          2.36.1               hea4e1c9_2    conda-forge
libarchive                3.5.2                hccf745f_1    conda-forge
libblas                   3.9.0           12_linux64_openblas    conda-forge
libcblas                  3.9.0           12_linux64_openblas    conda-forge
libcurl                   7.80.0               h2574ce0_0    conda-forge
libedit                   3.1.20191231         he28a2e2_2    conda-forge
libev                     4.33                 h516909a_1    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-ng                 11.2.0              h1d223b6_11    conda-forge
libgfortran-ng            11.2.0              h69a702a_11    conda-forge
libgfortran5              11.2.0              h5c6108e_11    conda-forge
libgomp                   11.2.0              h1d223b6_11    conda-forge
libiconv                  1.16                 h516909a_0    conda-forge
liblapack                 3.9.0           12_linux64_openblas    conda-forge
liblief                   0.11.5               h9c3ff4c_1    conda-forge
libnghttp2                1.43.0               h812cca2_1    conda-forge
libopenblas               0.3.18          pthreads_h8fe5266_0    conda-forge
libsodium                 1.0.18               h36c2ea0_1    conda-forge
libssh2                   1.10.0               ha56f1ee_2    conda-forge
libstdcxx-ng              11.2.0              he4da1e4_11    conda-forge
libxml2                   2.9.12               h885dcf4_1    conda-forge
libzlib                   1.2.11            h36c2ea0_1013    conda-forge
license-expression        1.2                        py_0    conda-forge
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
lzo                       2.10              h516909a_1000    conda-forge
markupsafe                2.0.1            py39h3811e60_1    conda-forge
msrest                    0.6.21             pyh44b312d_0    conda-forge
ncurses                   6.2                  h58526e2_4    conda-forge
numpy                     1.21.4           py39hdbf815f_0    conda-forge
oauthlib                  3.1.1              pyhd8ed1ab_0    conda-forge
openssl                   1.1.1l               h7f98852_0    conda-forge
packaging                 21.0               pyhd8ed1ab_0    conda-forge
patchelf                  0.13                 h58526e2_0    conda-forge
pcre2                     10.37                h032f7d1_0    conda-forge
perl                      5.32.1          1_h7f98852_perl5    conda-forge
pip                       21.3.1             pyhd8ed1ab_0    conda-forge
pkginfo                   1.7.1              pyhd8ed1ab_0    conda-forge
progressbar2              3.53.1             pyh9f0ad1d_0    conda-forge
psutil                    5.8.0            py39h3811e60_2    conda-forge
py-lief                   0.11.5           py39he80948d_1    conda-forge
pycosat                   0.6.3           py39h3811e60_1009    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pycrypto                  2.6.1           py39h3811e60_1006    conda-forge
pygithub                  1.55               pyh6c4a22f_0    conda-forge
pygments                  2.10.0             pyhd8ed1ab_0    conda-forge
pyjwt                     2.3.0              pyhd8ed1ab_0    conda-forge
pynacl                    1.4.0            py39h3811e60_3    conda-forge
pyopenssl                 21.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.5              pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1            py39hf3d152e_4    conda-forge
python                    3.9.7           hb7a2778_3_cpython    conda-forge
python-libarchive-c       3.1              py39hf3d152e_0    conda-forge
python-utils              2.5.6              pyh44b312d_0    conda-forge
python_abi                3.9                      2_cp39    conda-forge
pytz                      2021.3             pyhd8ed1ab_0    conda-forge
pyyaml                    6.0              py39h3811e60_3    conda-forge
rapidfuzz                 1.8.2            py39hde0f152_1    conda-forge
readline                  8.1                  h46c0cb4_0    conda-forge
requests                  2.26.0             pyhd8ed1ab_0    conda-forge
requests-oauthlib         1.3.0              pyh9f0ad1d_0    conda-forge
ripgrep                   13.0.0               habb4d0f_1    conda-forge
ruamel.yaml               0.17.17          py39h3811e60_0    conda-forge
ruamel.yaml.clib          0.2.2            py39h3811e60_2    conda-forge
ruamel.yaml.jinja2        0.2.4                      py_1    conda-forge
ruamel_yaml               0.15.80         py39h3811e60_1006    conda-forge
scrypt                    0.8.18           py39h3da14fd_0    conda-forge
setuptools                58.5.3           py39hf3d152e_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
smmap                     3.0.5              pyh44b312d_0    conda-forge
snowballstemmer           2.1.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.3                pyhd8ed1ab_0    conda-forge
sphinx                    4.2.0              pyh6c4a22f_0    conda-forge
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    2.0.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_0    conda-forge
sqlite                    3.36.0               h9cd32fc_2    conda-forge
stdlib-list               0.7.0                      py_2    conda-forge
tk                        8.6.11               h27826a3_1    conda-forge
toolz                     0.11.2             pyhd8ed1ab_0    conda-forge
tqdm                      4.62.3             pyhd8ed1ab_0    conda-forge
typing_extensions         3.10.0.2           pyha770c72_0    conda-forge
tzdata                    2021e                he74cb21_0    conda-forge
urllib3                   1.26.7             pyhd8ed1ab_0    conda-forge
vsts-python-api           0.1.22                     py_0    conda-forge
wheel                     0.37.0             pyhd8ed1ab_1    conda-forge
wrapt                     1.13.3           py39h3811e60_1    conda-forge
xz                        5.2.5                h516909a_1    conda-forge
yaml                      0.2.5                h516909a_0    conda-forge
zlib                      1.2.11            h36c2ea0_1013    conda-forge
zstd                      1.5.0                ha95c52a_0    conda-forge
```

```
→ pip list
Package                       Version
----------------------------- ---------
alabaster                     0.7.12
Babel                         2.9.1
beautifulsoup4                4.10.0
blinker                       1.4
boolean.py                    3.7
brotlipy                      0.7.0
certifi                       2021.10.8
cffi                          1.15.0
chardet                       4.0.0
charset-normalizer            2.0.0
colorama                      0.4.4
conda                         4.10.3
conda-build                   3.21.6
conda-package-handling        1.7.3
conda-smithy                  3.14.3
cryptography                  35.0.0
Deprecated                    1.2.13
docutils                      0.16
filelock                      3.3.2
gitdb                         4.0.9
GitPython                     3.1.24
glob2                         0.7
grayskull                     0.9.1
idna                          3.1
imagesize                     1.3.0
isodate                       0.6.0
Jinja2                        3.0.3
libarchive-c                  3.1
license-expression            1.2
MarkupSafe                    2.0.1
msrest                        0.6.21
numpy                         1.21.4
oauthlib                      3.1.1
packaging                     21.0
pip                           21.3.1
pkginfo                       1.7.1
progressbar2                  3.53.1
psutil                        5.8.0
pycosat                       0.6.3
pycparser                     2.21
pycrypto                      2.6.1
PyGithub                      1.55
Pygments                      2.10.0
PyJWT                         2.3.0
PyNaCl                        1.4.0
pyOpenSSL                     21.0.0
pyparsing                     3.0.5
PySocks                       1.7.1
python-utils                  2.5.6
pytz                          2021.3
PyYAML                        6.0
rapidfuzz                     1.8.2
requests                      2.26.0
requests-oauthlib             1.3.0
ruamel.yaml                   0.17.17
ruamel.yaml.clib              0.2.2
ruamel-yaml-conda             0.15.80
ruamel.yaml.jinja2            0.2.4
scrypt                        0.8.18
setuptools                    58.5.3
six                           1.16.0
smmap                         3.0.5
snowballstemmer               2.1.0
soupsieve                     2.3
Sphinx                        4.2.0
sphinxcontrib-applehelp       1.0.2
sphinxcontrib-devhelp         1.0.2
sphinxcontrib-htmlhelp        2.0.0
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          1.0.3
sphinxcontrib-serializinghtml 1.1.5
stdlib-list                   0.7.0
toolz                         0.11.2
tqdm                          4.62.3
typing-extensions             3.10.0.2
urllib3                       1.26.7
vsts                          0.1.22
wheel                         0.37.0
wrapt                         1.13.3
```

</details>

**Additional context**

xref: https://github.com/conda-forge/dask-sql-feedstock/pull/28

**Comment by marcelotrevisani** [comment]
indeed that is a bug, gonna take a look at it, thanks!

**Comment by jakirkham** [comment]
Thanks Marcelo! 😄

**Comment by marcelotrevisani** [comment]
Fixed with #277 
Thanks for reporting it :)
